README.md

```diff
@@ -127,26 +146,38 @@ Refinery validates its configuration on startup or when a configuration is reloa
 
 ### Configuration Query
 
-Because the configuration file formats (YAML) can sometimes be confusing to read and write, it may be valuable to check the loaded configuration by using one of the `/query` endpoints from the command line on a server that can access a refinery host.
+When using the standard configuration file formats, such as TOML and YAML, check the loaded configuration by using one of the `/query` endpoints from the command line on a server that can access a Refinery host.
```

Comment:
I don't want to refer to TOML. I took it out deliberately. If someone wants TOML they can get it from the query endpoint, but let's not point them to it.

Honestly, this feature is now largely obsolete since we have validation.

Reply:
```suggestion
Check the loaded configuration by using one of the `/query` endpoints from the command line on a server that can access a Refinery host.
```